packages/react-components/react-skeleton/src/components/Skeleton/useSkeletonStyles.ts

```diff
@@ -1,33 +1,121 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { SkeletonSlots, SkeletonState } from './Skeleton.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
 
 export const skeletonClassNames: SlotClassNames<SkeletonSlots> = {
   root: 'fui-Skeleton',
-  // TODO: add class names for all slots on SkeletonSlots.
-  // Should be of the form `<slotName>: 'fui-Skeleton__<slotName>`
+};
+
+export const skeletonGroupClassName = 'fui-Skeleton-Group';
+
+const skeletonWaveAnimation = {
+  from: {
+    backgroundPositionX: '300%',
+  },
+  to: {
+    backgroundPositionX: '0%',
+  },
+};
+
+const skeletonWaveAnimationRTL = {
+  from: {
+    backgroundPositionX: '0%',
+  },
+  to: {
+    backgroundPositionX: '300%',
+  },
+};
+
+const skeletonPulseAnimation = {
+  from: {
+    opacity: '1',
+  },
+  to: {
+    opacity: '0.4',
+  },
 };
 
 /**
  * Styles for the root slot
  */
-const useStyles = makeStyles({
+const useRootStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
-  },
+    '& [class^= "fui-Skeleton-"]': {
```

Comment:
This selector is [a quite heavy for browsers](https://github.com/microsoft/fluentui/blob/master/rfcs/react-components/styles-handbook.md#understanding-selector-complexity). If you would like to apply the same styles for different components, I suggest to create a shared hook with styles and reuse it.

Reply:
I have upated the selector